build.sbt

```diff
@@ -6,7 +6,7 @@ description := "sbt plugin for cassandra schema/data migrations using pillar (ht
 
 organization := "io.ino"
 
-version := "2.0.0"
+version := "2.1.0"
```

Comment:
Usually PRs should only contain functional changes and release version should not be touched. AFAICS there are no other changes in the queue, so you can leave it now.